src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+import java.lang.reflect.Array;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
```

Comment:
Will it be better to add javadoc to all public classes and methods? The same applies to all other public classes and methods.